src/modules/cache-client/cache-client.types.ts

```diff
@@ -1,12 +1,11 @@
-import { SiweMessage } from 'siwe';
 import { AssetHistoryEventType } from '../assets';
 
 export enum Order {
   'ASC' = 'ASC',
   'DESC' = 'DESC',
 }
 
-export type SiweOptions = Pick<SiweMessage, 'domain'>;
+export type SiweOptions = { domain: string; uri?: string };
```

Comment:
When we have a moment, we should add `uri` and `domain` to the example in the README, IMO https://github.com/energywebfoundation/iam-client-lib#2-connect-to-the-cache-server